specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -12434,6 +12478,46 @@
           }
         }
       }
+    },
+    "TieringPolicy": {
+      "description": "Tiering Policy.",
+      "type": "object",
+      "properties": {
+        "tieringMode": {
+          "description": "Tiering Mode to control automatic tiering of recovery points",
+          "enum": [
+            "Invalid",
+            "TierRecommended",
```

Comment:
>TierRecommended [](http://example.com/codeflow?start=13&length=15)

What is the meaning of each of these modes? Can you provide the definition in comments?

Reply:
TierRecommended: Tier all recovery points recommended to be tiered
TierAfter: Tier all recovery points after a fixed period, as specified in duration+durationType below.

Reply:
Could u add this definition to the description in the swagger

Reply:
Sure, will add.

Reply:
Added